### DISCUSSION TITLE
Tutorial for how to install is honestly complete garbage. I've tried so many times I'm about to just give up on this.

### DISCUSSION THREAD

**Discussion by ambitendencies** [flowershow/flowershow#342] [General and Help]
I don't understand how this is so complicated for me to setup when I've done Next.js deploys several times before. The instructions are honestly just trash for this. You just say run npm flowershow build and then just say vague things like oh your site files are located under .flowershow/.next and leave it at that. I've tried deploying to vercel and netlify from the .next folder, from the .flowershow folder, and from the parent folder. They all give 404 errors. What the hell are you supposed to do?

**Comment by danymat** [comment]
Well I have opened an issue for my installation problems too, but I feel like the way you shared it could be view as an aggressive rant. I don't think developers will want to fix your issue if you write like this

**Comment by PhilippeduPreez** [comment]
@ambitendencies Did you check that your site works locally with `npx flowershow@latest preview` ? If that works try using `npx flowershow@latest export` and deploy the folder `.flowershow/out` . Please let know if you are still getting 404 errors after this.
